### PR TITLE
Adds bounty name, value and holder to bounty cube description

### PR DESCRIPTION
**Pull request by cacogen**
## About The Pull Request

Also adds the station time they were created and renames Bounty Cube to "[value] cr bounty cube".

## Why It's Good For The Game

It's useful and interesting for cargo to know who is completing bounties, how much they're worth and what they're for. It also motivates cargo to ship cubes when the value, creator and creation time are known to them.

## Changelog
:cl: cacogen
add: Bounty cubes now have the bounty's name, value, fulfiller and creation time in the description
tweak: The Bounty Cube is now known as the "[value] cr bounty cube"
/:cl:

### CODE REVIEW

**Comment by Losenis** [comment]
Isn't this redundant given how they're wrapped and tagged?

**Comment by cacogen** [comment]
Most of the time they aren't wrapped and tagged. Someone said last night when I was playing cargo that that doesn't work anymore (which may or may not be true).

File: code/game/machinery/civilian_bounties.dm [hunk 1]
```diff
@@ -95,11 +95,15 @@
...
 		//Pay for the bounty with the ID's department funds.
-		status_report += "Bounty Completed! Please send your completed bounty cube to cargo for your automated payout shortly."
+		status_report += "Bounty completed! Please give your bounty cube to cargo for your automated payout shortly."
 		inserted_scan_id.registered_account.reset_bounty()
 		SSeconomy.civ_bounty_tracker++
 		var/obj/item/bounty_cube/reward = new /obj/item/bounty_cube(drop_location())
 		reward.bounty_value = curr_bounty.reward
+		reward.bounty_name = curr_bounty.name
+		reward.bounty_holder = inserted_scan_id.registered_name
+		reward.name = "\improper [reward.bounty_value] cr" + " [reward.name]"
```
Comment (Jared-Fogle) [inline code/game/machinery/civilian_bounties.dm:105]:
```suggestion
		reward.name = "\improper [reward.bounty_value] cr [reward.name]"
```

Comment (cacogen) [reply code/game/machinery/civilian_bounties.dm:105]: Hmm that was dumb as all fuck

**Comment by ArcaneMusic** [comment]
> Isn't this redundant given how they're wrapped and tagged?



> Most of the time they aren't wrapped and tagged. Someone said last night when I was playing cargo that that doesn't work anymore (which may or may not be true).

For the millionth time:
You do not need to wrap and tag bounty cubes.
They already have a price tag component.
You JUST need to put them on the shuttle with no processing.

**Comment by cacogen** [comment]
People were wrapping and tagging them because it gave them a higher percentage. I'm not sure if that still works or not.

**Pull request merged**